Sources/KlaviyoSwift/Models/Event.swift

```diff
@@ -11,12 +11,15 @@ import KlaviyoCore
 
 public struct Event: Equatable {
     public enum EventName: Equatable {
-        case OpenedPush
         case OpenedAppMetric
         case ViewedProductMetric
         case AddedToCartMetric
         case StartedCheckoutMetric
         case CustomEvent(String)
+
+        internal static var _openedPush: EventName {
```

Comment:
Neat. Any chance you tested this with `client/events` and confirmed that it works? I.E we should we seeing a canonical opened push event in the profile for which we are tracking the open. Ex: 

![image](https://github.com/user-attachments/assets/d6c5d0de-782d-4580-b067-0e41e7dad30e)

Reply:
Yes, just validated that we're successfully tracking the Opened Push event

Reply:
Thanks for checking.